lib/utils/setup_appwrite.dart

```diff
@@ -28,6 +29,10 @@ void main() async {
       collectionId: 'users', key: 'email', xrequired: true);
   await db.createStringAttribute(
       collectionId: 'users', key: 'id', size: 256, xrequired: true);
-
   print("Collection created");
+
+  // Creating a new Bucket to store Profile Photos
+  await storage.createBucket(
```

Comment:
Since with the introduction of Buckets in version 0.13.x , we also need to create one from the server side. You can either create a new bucket from the dashboard or with this script, it will now create one from you. 

NOTE: You can change the Bucket ID and name given that you make those changes in other places too